index.bs

```diff
@@ -2031,20 +2002,39 @@ Some of the existing manifest files include
 * <a href="https://wicg.github.io/origin-policy/">Origin Policy</a> which is used to set security policies.
 
 We encourage people to think of manifest files as extensible.
```

Comment:
Not sure what this means. Either they're extensible or they're not extensible.

Reply:
"We encourage people to extend existing manifest files." as proposed replacement text.